modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -40,7 +40,7 @@ export default {
   },
   computed: {
     editLabel () {
-      return `Edit ${this.field.label}`;
+      return `Edit ${this.$t(this.field.label)}`;
```

Comment:
The whole thing needs l10n! See how other strings with interpolation are set up.

Reply:
i.e. no use if we don't l10n "Edit"

Reply:
oh roit